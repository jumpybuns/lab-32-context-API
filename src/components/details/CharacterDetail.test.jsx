import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CharacterDetail from './CharacterDetail';

describe('CharacterDetail component', () => {
  afterEach(() => cleanup());
  it('renders CharacterDetail', () => {
    const { asFragment } = render(
      <CharacterDetail
        name="Morty Smith"
        image="https://rickandmortyapi.com/api/character/avatar/2.jpeg"
        status="alive"
        species="human"
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
