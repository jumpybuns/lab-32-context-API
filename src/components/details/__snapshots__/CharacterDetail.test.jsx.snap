// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterDetail component renders CharacterDetail 1`] = `
<DocumentFragment>
  <figure
    data-testid="character"
  >
    <img
      alt="Morty Smith"
      src="https://rickandmortyapi.com/api/character/avatar/2.jpeg"
    />
    <figcaption>
      <p>
        Morty Smith
      </p>
      <p>
        alive
      </p>
      <p>
        human
      </p>
    </figcaption>
  </figure>
</DocumentFragment>
`;
